library/ssl_tls13_server.c

```diff
@@ -1151,6 +1151,34 @@ static int ssl_tls13_write_encrypted_extensions( mbedtls_ssl_context *ssl )
     return( ret );
 }
 
+/*
+ * Handler for MBEDTLS_SSL_SERVER_CERTIFICATE
+ */
+static int ssl_tls13_write_server_certificate( mbedtls_ssl_context *ssl )
+{
+#if defined(MBEDTLS_KEY_EXCHANGE_WITH_CERT_ENABLED)
+    int ret = mbedtls_ssl_tls13_write_certificate( ssl );
```

Comment:
On client side it is possible to send an empty Certificate message but less on server side I'd say even if it does not seem to be explicitly stated in the spec. Thus we probably should check that we have an available certificate here: `mbedtls_ssl_own_cert( ssl ) != NULL` and return in error otherwise (handshake failure I'd say).